### PR TITLE
Docs for query and session pool + remove crt

### PR DESCRIPTION
**Pull request by chbatey**

### CODE REVIEW

File: journal/src/main/scala/akka/persistence/spanner/javadsl/SpannerReadJournal.scala [hunk 1]
```diff
@@ -17,10 +25,34 @@ object SpannerReadJournal {
...
+    with CurrentEventsByPersistenceIdQuery
+    with EventsByPersistenceIdQuery {
   override def currentEventsByTag(tag: String, offset: Offset): Source[EventEnvelope, NotUsed] =
     delegate.currentEventsByTag(tag, offset).asJava
 
   override def eventsByTag(tag: String, offset: Offset): Source[EventEnvelope, NotUsed] =
     delegate.eventsByTag(tag, offset).asJava
+
+  override def persistenceIds(): Source[String, NotUsed] =
+    delegate.persistenceIds().asJava
```
Comment (chbatey) [inline journal/src/main/scala/akka/persistence/spanner/javadsl/SpannerReadJournal.scala:40]: Oops :)

File: docs/src/main/paradox/tuning-session-pool.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+# gRPC client and Session Pool
+
+The plugin uses the Spanner gRPC API. When interacting with Spanner over gRPC a session pool needs to be maintained.
+Each spanner transaction, read or write, needs to be done in the context of a session.
+
+The default session pool size is 5, which limits the number of concurrent reads and writes to Spanner.
+This value will need to be increased if you use-case has a large number of concurrent writes or reads.
+
+Sessions are kept alive indefinitely. A dynamic session pool is planned for future versions of the plugin so.
```
Comment (johanandren) [inline docs/src/main/paradox/tuning-session-pool.md:9]: Maybe skip the "planned" sentence, we have an issue.

File: docs/src/main/paradox/tuning-session-pool.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+The maximum value is 100, set by Spanner. Future versions of the plugin may create multiple gRPC clients to allow
+more than 100 sessions at a time.
```
Comment (johanandren) [inline docs/src/main/paradox/tuning-session-pool.md:12]: Perhaps link to issue.

**Comment by chbatey** [comment]
I think for most ppl it will just work

**Pull request merged**